File: assets/AssignmentTwo/JS/SeriesOne.js
let n = +prompt("How many number you enter: ", "");

let series = '';
for (let i = 1; i <= n; i++) {
  series += (1 / i).toFixed(2) + ' ';
}

console.log("\n\nSeries of : (1/1 + ½ + ⅓ +...................+1/n) is");
console.log("------------------------\n");
console.log(series);

File: assets/AssignmentTwo/C/SeriesOne.c
#include <stdio.h>

int main(void) {
  int i, n;

  printf("How many number you enter: ");
  scanf("%d", &n);

  printf("\n\nSeries of : (1/1 + ½ + ⅓ +...................+1/n) is \n");
  printf("------------------------\n\n");
  for (i = 1; i <= n; i++) {
    printf("%.2lf ", (double)1 / i);
  }
}